From 388ced2619c629a92bf17191be8186ee1ef96861 Mon Sep 17 00:00:00 2001
From: Biju Chakkachattil <bijux.chakkachattil@intel.com>
Date: Wed, 6 May 2015 08:45:28 +0530
Subject: [PATCH] BQ24192: Resume charging upon exiting battery under
 temperature condition.

In cos mode when temperature goes below 0, battery charging stops.
Upon exiting from under temperature condition, the charging was
not resumed. This patch will send the health status event up on
exiting the under temperature condition to resume the battery charging.

Change-Id: I14d62c3e32ed7bf757e7e4ee840754799aa8235c
Signed-off-by: Biju Chakkachattil <bijux.chakkachattil@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-31212
---
 drivers/power/bq24192_charger.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/power/bq24192_charger.c b/drivers/power/bq24192_charger.c
index 336968a38157..e4c8eaa61b16 100644
--- a/drivers/power/bq24192_charger.c
+++ b/drivers/power/bq24192_charger.c
@@ -1925,6 +1925,7 @@ sched_task_work:
 
 	curr_health = bq24192_get_battery_health();
 	if (prev_health != curr_health) {
+		power_supply_changed(fg_psy);
 		power_supply_changed(&chip->usb);
 		dev_warn(&chip->client->dev,
 			"%s health status  %d",
-- 
1.9.1

